### PR TITLE
Update manage-deployment-of-add-ins.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about integrated apps and only Global admins can access it.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4159

File: microsoft-365/admin/manage/manage-deployment-of-add-ins.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Before you begin, see [Determine if Centralized Deployment of add-ins works for
 2. Select **Deploy Add-in** at the top of the page, and then select **Next**.
  
     > [!NOTE]
-    > The admin center is getting updated to deployment experience with Integrated Apps . If you don't see the above steps, go to Centralized Deployment section by going to **Settings** > **Integrated apps**. On the top of the **Integrated apps** page, choose **Add-ins**.
+    > The admin center is getting updated to deployment experience with Integrated Apps .The Integrated Apps is supposed to be visible for Global administrator only, while for others the old experience still exist. If you don't see the above steps, go to Centralized Deployment section by going to **Settings** > **Integrated apps**. On the top of the **Integrated apps** page, choose **Add-ins**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-deployment-of-add-ins.md]:
```suggestion
    > The admin center is getting updated to deployment experience with Integrated Apps. Integrated Apps is only visible to Global administrators, while for others the old experience still exists. If you don't see the above steps, go to the Centralized Deployment section by going to **Settings** > **Integrated apps**. On the top of the **Integrated apps** page, choose **Add-ins**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@scotvidican Please assign it to the right person for the merge. Thanks.

**Pull request merged**